src/containers/EditorsColumn.js

```diff
@@ -41,6 +44,33 @@ function mapDispatchToProps(dispatch) {
     onRequestedLineFocused() {
       dispatch(editorFocusedRequestedLine());
     },
+
+    onEditorCursorChange(source, cursor, language) {
+      dispatch(
+        currentCursorChanged(
+          source,
+          cursor,
+          language,
+        ),
+      );
+    },
+
+    onEditorBlurred() {
+      dispatch(
+        editorBlurred(),
```

Comment:
Should this still contain a `language` argument? As it is I think we rely on ACE dispatching the `blur` event before the `focus` event in the case where the user clicks from one editor to another.